queue/models.py

```
@@ -17,7 +18,7 @@ class Submission(models.Model):
     '''
     # Submission 
     requester_id     = models.CharField(max_length=CHARFIELD_LEN_SMALL) # ID of LMS
-    lms_callback_url = models.CharField(max_length=CHARFIELD_LEN_SMALL, db_index=True)
+    lms_callback_url = models.CharField(max_length=CHARFIELD_LEN_MEDIUM, db_index=True)
```

Ugh, I feel like I'm flashing back on this. The following is vague recollection, and you'd have to look into it to be sure, but... I think it silently truncates this, even in prod. If I remember correctly, what happens is that this field gets truncated, but the actual contents are redundantly written as part of the `xqueue_header`, and it's the value from `xqueue_header` that's used to actually do the post back. So the primary use of the `lms_callback_url` is for doing searches and freaking out anyone examining the database.